src/Microsoft.DotNet.ServiceFabric.ServiceHost/DelegatedStatefulService.cs

```diff
@@ -85,18 +85,45 @@ protected override IEnumerable<ServiceReplicaListener> CreateServiceReplicaListe
         }
 
         protected override async Task RunAsync(CancellationToken cancellationToken)
+        {
+            await Task.WhenAll(RunSchedule(cancellationToken),
+                RunAsyncLoop(cancellationToken));
+        }
+
+        private async Task RunSchedule(CancellationToken cancellationToken)
```

Comment:
These function names are backwards. This one is calling RunAsync, but called RunSchedule. Please switch the names.